### PR TITLE
Added teams section carousel in the home page.

### LINKED ISSUE #69: A teams section carousel in the home page.
I am GSSOC'21 Participant. I would like to make a slidebar that would give information about team members on a auto play loop. It would look good on the webpage.
I would like to work on this issue please assign this to me.

### PR DESCRIPTION
**Pull request by Mrinal81**
Fixes #69 
Hi, I have added the slidebar for team member info in about session. Please review the changes that i have done.
Thank you.

### CODE REVIEW

**Comment by Mrinal81** [comment]
@Shruti3004  I made a pull request. KIndly review my pull request.

File: css/style.css [hunk 1]
```diff
@@ -232,16 +232,33 @@ header {
 
 /******************* ABOUT ***********************/
 
+
 .about {
-    padding: 150px 0px;
+  height: 800px;
+  padding: 6.8% 0% 7%;
+  text-align: center;
+
```
Comment (harikesh409) [inline css/style.css:240]: remove unnecessary blank lines.

File: index.html [hunk 2]
```diff
@@ -132,11 +132,144 @@
     <!--social media sidebae ends-->
 
     <!-- about starts -->
-    <section id="about" class="about">
-        <div class="container">
-            <h1 class="text-center">About Us</h1>
-            <hr>
 
+    <section id="about" class="about" style="background-color: 		#A9A9A9;">
```
Comment (harikesh409) [inline index.html:136]: change the background color it is not in sync with the websites color scheme and add the property in the style.css file and not inline.

File: index.html [hunk 3]
```diff
@@ -132,11 +132,144 @@
...
+     <div id="about-carousel" class="carousel slide" data-ride="carousel" data-interval="3000" data-pause="hover">
+       <div class="carousel-inner">
+         <div class="container">
+           <div class="carousel-item active">
+             <h1 class="text-center">About Us</h1>
```
Comment (harikesh409) [inline index.html:141]: The heading should be Team and shouldn't be a part of the carousel.

File: index.html (hunk 2 above)
Comment (harikesh409) [reply index.html:136]: Instead of merging with the about section create a new section that says team and use that. Leave the about us section as it is.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Pull the latest changes from master and merge your code.

**Comment by Mrinal81** [comment]
@harikesh409  I have made the changes as you asked can you please review it. I am attaching the image as well as video of it. I have doubt  about background color in Team section so could you see that image also.....
Screenshot from 2021-04-01 23-47-08
Screenshot from 2021-04-01 23-48-13
Screenshot from 2021-04-01 23-47-24

https://user-images.githubusercontent.com/76482022/113340214-353cba00-9349-11eb-93b3-8dcb9d82eaaf.mp4

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Code for changes present in Screenshots are there. Did you push the changes to this PR only? And also the code is not in sync with the master branch kindly pull the latest changes.

**Comment by Mrinal81** [comment]
@harikesh409 Done with all the changes. Please have a look 
 If everything is ok then please add GSSOC21 and Level  Tag....

**Review by harikesh409 — APPROVED** [review]

**Comment by harikesh409** [comment]
> @harikesh409 Done with all the changes. Please have a look
> If everything is ok then please add GSSOC21 and Level Tag....

Code looks good but the footer is not sync with the main branch kindly pull and merge.

**Comment by Mrinal81** [comment]
@harikesh409 ok

**Comment by Mrinal81** [comment]
@harikesh409 How i can sync footer with main branch... can you tell me because i don't no such things i am new on github...

**Comment by Mrinal81** [comment]
@harikesh409 please check once..

**Comment by harikesh409** [comment]
> @harikesh409 please check once..

Looks good lets wait for @ravikr126 review.

**Review by ravikr126 — APPROVED** [review]
Good everything is good

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**